src/core/reopt_inputs.jl

```diff
@@ -396,8 +396,8 @@ function setup_tech_inputs(s::AbstractScenario)
     end
 
     if "Boiler" in techs.all
-        setup_boiler_inputs(s, max_sizes, min_sizes, existing_sizes, cap_cost_slope, 
-            boiler_efficiency, production_factor, fuel_cost_per_kwh)
+        setup_boiler_inputs(s, max_sizes, min_sizes, existing_sizes, cap_cost_slope, boiler_efficiency,
```

Comment:
I'm curious how this wasn't causing any big issues before?

Reply:
Good question! We were putting in the "existing_sizes" array for the argument where the setup_boiler_function() was expecting "om_cost_per_kw". Since the function call had the expected number of arguments, it didn't error there. Then it was actually assigning existing_sizes["Boiler"] = s.boiler.om_cost_per_kw, when it ran the line om_cost_per_kw["Boiler"] = s.boiler.om_cost_per_kw. I just tested this with the develop branch and verified inputs.existing_sizes["Boiler"] = [a non zero value] when I input a non-zero value for Boiler.om_cost_per_mmbtu_per_hour (gets converted into _per_kw). I guess having an "existing_size" for Boiler didn't cause any other issues - it still passes the test with or without a non-zero value.